src/main/java/com/github/algobot76/surabaya/util/StaticAnalysisVisitor.java

```diff
@@ -98,8 +110,13 @@ public void visit(MethodDeclaration n, Project project) {
 
 	@Override
 	public void visit(ConstructorDeclaration n, Project project) {
+		if (n.getRange().isEmpty()) {
+			return;
+		}
+		Range range = n.getRange().get();
+		String src = getSrcFromRange(range);
 		Constructor constructor = new Constructor(n.getName().getIdentifier().trim(),
-				getAccessModifier(n.getModifiers()));
+				getAccessModifier(n.getModifiers()), src);
```

Comment:
The easiest way to get the code of the constructor is to call the `toString` method on `n`.